examples/integers/src/main.hs

```diff
@@ -0,0 +1,8 @@
+{-# LANGUAGE ForeignFunctionInterface #-}
+
+import Foreign.C.Types (CUInt(..))
```

Comment:
You're right, `Word32` would be the right choice here. (For some reason, I had _thought_ `CUInt` was guaranteed to be 32 bits, but apparently that's not the case.) I've updated accordingly.